app/assets/javascripts/govuk_publishing_components/components/accordion.js

```diff
@@ -80,7 +86,11 @@ window.GOVUK.Modules.GovukAccordion = window.GOVUKFrontend.Accordion;
 
   // Find the parent accordion section for the given id and open it
   GemAccordion.prototype.openForAnchor = function (hash) {
-    var target = document.getElementById(hash)
+    if (hash.indexOf(':') !== -1) {
+      hash = hash.replace(':', '\\:')
+    }
+
```

Comment:
Another tiny suggestion here - but I think the code above looks quite similar to the code here:

```
GemAccordion.prototype.openForAnchor = function (hash) {
  if (hash.indexOf(':') !== -1) {
    hash = hash.replace(':', '\\:')
  }
  ...
```

```
GemAccordion.prototype.openByAnchorOnLoad = function () {
  if (!window.location.hash) return
  var splitHash = window.location.hash.split('#')[1]
  var targetID = window.location.hash

  if (targetID.indexOf(':') !== 1) {
    targetID = targetID.replace(':', '\\:')
  }
  ...
```
Maybe we could have a separate function that could work for both?